plugin/src/main/java/org/wildfly/plugin/provision/ChannelMavenArtifactRepositoryManager.java

```diff
@@ -84,18 +87,28 @@ public void resolve(MavenArtifact artifact) throws MavenUniverseException {
         try {
             resolveFromChannels(artifact);
         } catch (UnresolvedMavenArtifactException ex) {
-            // unable to resolve the artifact through the channel.
-            // if the version is defined, let's resolve it directly
-            if (artifact.getVersion() == null) {
+            if (originalVersionResolution) {
+                log.warn("Resolution of artifact " + artifact.getGroupId() + ":" +
+                        artifact.getArtifactId() + " failed. Using original version.");
+                // unable to resolve the artifact through the channel.
+                // if the version is defined, let's resolve it directly
+                if (artifact.getVersion() == null) {
+                    log.warn("No version provided.");
```

Comment:
Should this actually be a error since it throws an exception?

Reply:
Good catch James, Fixed!